ssn/ssn_separated/ssn.ttl

```diff
@@ -220,16 +203,16 @@ ssn:hasProperty rdf:type owl:ObjectProperty ;
 
 ###  http://www.w3.org/ns/ssn/hasSubSystem
 ssn:hasSubSystem rdf:type owl:ObjectProperty ;
-                 rdfs:comment "Relation between a system and its component parts (has-part)." ;
+                 rdfs:comment "Relation between a System and its component parts (has-part)." ;
                  rdfs:isDefinedBy "http://www.w3.org/ns/ssn" ;
-                 rdfs:label "has subsystem" ;
-                 rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Model#System" .
+                 rdfs:label "has subsystem" .
 
 
 ###  http://www.w3.org/ns/ssn/hasSurvivalProperty
 ssn:hasSurvivalProperty rdf:type owl:ObjectProperty ;
                         rdfs:subPropertyOf ssn:hasProperty ;
-                        rdfs:comment "Relation from a SurvivalRange of a System to a Property describing the survival range of the system. For example, to the temperature range that a system can withstand before being considered damaged." ;
+                        rdfs:comment "Relation from a SurvivalRange of a System to a Property describing the survival qualities of the system." ;
```

Comment:
Relation from a SurvivalRange of a System to a SurvivalProperty describing the extent of the System's useful life.